server/middlewares/UserValidation.js

```diff
@@ -72,7 +171,7 @@ class UserValidation {
         if (foundEmail) {
           return res.status(409).json({
             errors: {
-              message: ['email is already in use']
+              email: ['email is already in use']
             }
```

Comment:
I don't think we should be returning an error inside this function, it makes it only useful for the signup route (or at least a route that requires the email to not be in use). 
The log in route, for example, can't use this function since we want the email to exist in our db to log in.

I suggest we return true or false, then custom functionality/error can be written from where it's being called.